HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -17,6 +17,19 @@
 #                     pset.minGoodStripCharge = cms.PSet(refToPSet_ = cms.string('HLTSiStripClusterChargeCutNone'))
 #     return process
 
+
+def customiseForPR22293(process):
+    for producer in producers_by_type(process, "HBHEPhase1Reconstructor"):
+        producer.algorithm.chiSqSwitch = cms.double(15.0)
+        producer.saveEffectivePedestal = cms.bool(True)
+        if (producer.algorithm.useMahi == cms.bool(True)):
+            producer.algorithm.applyTimeConstraint = cms.bool(False)
+            producer.algorithm.applyPedConstraint = cms.bool(False)
+        if hasattr(producer.algorithm.timeSlewPars):
```

Comment:
`hasattr(producer.algorithm,"timeSlewPars")`
producer.algorithm is the object, timeSlewPars is the name of the attribute (sent as a string)

Reply:
both the if hasattr(producer.algorithm.timeSlewPars) and if producer.algorithm.timeSlewPars did the correct removal, I checked in the dump do you want to me move to your sintax ?

Reply:
The if statement is meant to protect against trying to access `producer.algorithm.timeSlewPars` if the attribute *doesn't* exist. What's there right now won't do that.

Reply:
ok, done